### ISSUE TITLE
Improve dropdown

### ISSUE THREAD

**Issue by mkaring** [nifty-gui/nifty-gui#49]
The dropdown control has currently some limits. For 1.3.2 I suggest to improve this element.

The keyboard focus should be placed on the selected element once the popup opens. Also the listbox should be scrolled to the selected element. These are the most important things.
